thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/DetectionJobResource.java

```diff
@@ -615,9 +627,16 @@ public Response tuneAlertFilter(@PathParam("functionId") long id,
     }
 
     // write to DB
-    autotuneConfig.setFunctionId(id);
-    autotuneId = DAO_REGISTRY.getAutotuneConfigDAO().save(autotuneConfig).toString();
-    return Response.ok(autotuneId).build();
+    List<Long> autotuneIds = new ArrayList<>();
+    for (long functionId : anomalyFunctionIds) {
+      autotuneConfig.setId(null);
+      autotuneConfig.setFunctionId(functionId);
+      AnomalyFunctionDTO anomalyFunctionSpec = DAO_REGISTRY.getAnomalyFunctionDAO().findById(functionId);
+      autotuneConfig.setAlertFilter(alertFilterFactory.fromSpec(anomalyFunctionSpec.getAlertFilter()));
```

Comment:
setConfiguration or setAlertFilter? Only use setConfiguration can be put to database. If want to record to db, please use setConfiguration.

Reply:
Please remove this line if we don't want to save current alert filter anymore. But better have it logged. Thanks!